git_aggregator/repo.py

```diff
@@ -33,7 +33,7 @@ class Repo(object):
 
     _git_version = None
 
-    def __init__(self, cwd, remotes, merges, target,
+    def __init__(self, cwd, remotes, merges, target, fetch_all=False,
```

Comment:
To preserve backward compatibility it's better to add new optional parameters at the end.